src/Html/Rules/NoTextEmptyString.elm

```diff
@@ -0,0 +1,106 @@
+module Html.Rules.NoTextEmptyString exposing (error, rule)
```

Comment:
The user has no use for this `error` value, so it's better to hide it. I usually duplicate the error details in my tests and in my rule, it's not a big deal.